python/cudf/cudf/core/dataframe.py

```diff
@@ -4450,6 +4450,97 @@ def skew(
             **kwargs,
         )
 
+    def clip(self, lower=None, upper=None, inplace=False):
```

Comment:
Lets add axis argument to be consistent with Pandas. If someone passes an axis we don't support then lets throw a `NotImplementedError` for now.